client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -293,7 +299,7 @@ export class DraggablePanelComponent implements OnInit, OnChanges {
     }
   }
 
-  zoomIn(level?: number) {
+  zoomIn(level?: number, polygon?: boolean) {
```

Comment:
Notice that you call it above as `zoomIn(null, true)`. You could simplify that by reversing the inputs
`zoomIn(polygon?: boolean, level?: number)` then change the calls to `zoomIn(true)`

Not really worth changing, just a thought on keeping the inputs simple